books-api/Book Copies ADR.md

```diff
@@ -136,11 +136,11 @@ erDiagram
     }
     BooksCopiesReadingHistory {
         long id PK
-        long bookCopyId FK
-        long holderEmployeeId
+        long copyId FK
+        long readerEmployeeId
         timestamptz takenAtUtc
-        timestamptz returnedAtUtc "Nullable"
-        string progressOfReading "NotReadAtAll|ReadPartially|ReadEntirely"
+        timestamptz sheduledReturnAtUtc
```

Comment:
Is it a required action to decide when you will return it? This is also a date (below there are more details). Rename to `sheduledReturnDate`. Check which types exist for dates in PostgreSQL and .NET (we did it before somewhere).